department-for-transport-BODS/create-data
# feat: CFD-457-460-view-flat-fare-product-details

# Description

-    Create a page that displays all key elements of a fare.

# Testing instructions

-   Create fares for:
1. Set of services
2. Carnet set of services
3. Geo zone
4. Carnet Geozone
5. School set of services
6. Carnet School Set of Services
7. Period Geozone
8. Period Geozone Carnet
9. Period Set of Services
10. Period set of services carnet
11. Hybrid Period
12. Hybrid Period Carnet
13. School Set of services
14. School set of services carnet
15. Multi Op Set of Services period
16. Multi Op Geo zone period

- Click through each fare on http://localhost:5555/products/otherProducts and check that each field is correct referencing the jira tickets

# Type of change
